diff --git a/dist/chunks/cli-api.B7PN_QUv.js b/dist/chunks/cli-api.B7PN_QUv.js
index 598757462e99e6a3c7159f6a4efb0729939e8a32..c37abdffbfcb434613fad4c1a211804c3b2411d7 100644
--- a/dist/chunks/cli-api.B7PN_QUv.js
+++ b/dist/chunks/cli-api.B7PN_QUv.js
@@ -5116,7 +5116,7 @@ function isValidApiRequest(config, req) {
 	try {
 		const token = url.searchParams.get("token");
 		if (token && crypto.timingSafeEqual(Buffer.from(token), Buffer.from(config.api.token))) return true;
-	} 
+	}
 	// an error is thrown when the length is incorrect
 catch {}
 	return false;
@@ -6508,7 +6508,7 @@ class Logger {
 		this._highlights.clear();
 		this.addCleanupListeners();
 		this.registerUnhandledRejection();
-		if (this.outputStream.isTTY) this.outputStream.write(HIDE_CURSOR);
+		if (this.outputStream.isTTY && !process.env.CI) this.outputStream.write(HIDE_CURSOR);
 	}
 	log(...args) {
 		this._clearScreen();
@@ -8065,7 +8065,7 @@ class Pool {
 				this.exitPromises.push(runner.stop({ force: resolver.isRejected }).then(() => clearTimeout(id)).catch((error) => this.logger.error(`[vitest-pool]: Failed to terminate ${task.worker} worker for test files ${formatFiles(task)}.`, error)));
 			}
 			this.freeWorkerId(poolId);
-		} 
+		}
 		// This is mostly to avoid zombie workers when/if Vitest internals run into errors
 catch (error) {
 			return resolver.reject(error);
